examples/data/e2e_tests/data/hdfs/std/std_test/2019/11/27/v1/data.json

```diff
@@ -0,0 +1,5 @@
+    
+    {"property":"all_filled","string_type": "nazdarek","boolean_type":"TRUE","binary_type": "binary","byte_type":127,"timestamp_type":"25/08/2067  14:56:22 UTC","integer_type":2147483647,"date_type":"2019-12-24","decimal_type":1.9,"long_type":9223372036854775807,"double_type":17523.6,"short_type":32767,"float_type":3529.2, "struct_type":{"boolean_type":"TRUE","string_type": "nazdarek","byte_type":127,"timestamp_type":"25/08/2067  14:56:22 UTC","integer_type":2147483647,"date_type":"2019-12-24","decimal_type":1.9,"long_type":9223372036854775807,"double_type":17523.6,"short_type":32767,"binary_type": "binary","float_type":3529.2,"boolean_type_array":["TRUE", "TRUE", "FALSE"],"byte_type_array":[127, 126, 110],"timestamp_type_array": ["25/08/1955  14:56:22 UTC","25/08/2067  14:56:22 UTC","25/02/2020  18:00:22 UTC"], "integer_type_array":[54345, 55432, 2345],"date_type_array": ["2019-12-24", "2019-12-24", "2019-12-24"],"decimal_type_array": [1.5, 2.6, 5.556],"long_type_array": [9223372036854775807, 9223372036854775777, 223372036854775807],"double_type_array": [17523.76, 1752.6, 17.523],"short_type_array": [32767, 32766, 32765],"binary_type_array": ["true","false"],"float_type_array": [529.2, 3529.2],"string_type_array": ["ahoj", "nazdar", "cus"]},"boolean_type_array":["TRUE", "TRUE", "FALSE"],"byte_type_array":[127, 126, 110],"timestamp_type_array": ["25/08/1955  14:56:22 UTC","25/08/2067  14:56:22 UTC","25/02/2020  18:00:22 UTC"],"integer_type_array":[54345, 55432, 2345],"date_type_array": ["2019-12-24", "2019-12-24", "2019-12-24"],"decimal_type_array": [1.5, 2.6, 5.556],"long_type_array": [9223372036854775807, 9223372036854775777, 223372036854775807],"double_type_array": [17523.76, 1752.6, 17.523],"short_type_array": [32767, 32766, 32765],"binary_type_array": ["true","false"],"float_type_array": [529.2, 3529.2],"string_type_array": ["ahoj", "nazdar", "cus"],"struct_type_array":[{"boolean_type":"TRUE","string_type": "nazdarek","byte_type":127,"timestamp_type":"25/08/2067  14:56:22 UTC","integer_type":2147483647,"date_type":"2019-12-24","decimal_type":1.9,"long_type":9223372036854775807,"double_type":17523.6,"short_type":32767,"binary_type": "binary","float_type":3529.2},{"boolean_type":"TRUE","string_type": "nazdarek","byte_type":127,"timestamp_type":"25/08/2067  14:56:22 UTC","integer_type":2147483647,"date_type":"2019-12-24","decimal_type":1.9,"long_type":9223372036854775807,"double_type":17523.6,"short_type":32767,"binary_type": "binary","float_type":3529.2}]}
```

Comment:
I don't know if the "nazdarek, ahoj, cus" data are particularly suitable, but it gets the job done, I suppose. 😆

Reply:
it's a remnant of history, I'll fix it in one of the next versions. Thank you.

Reply:
I like it. Little Czech touch 😉